install-panel.sh

```diff
@@ -804,6 +834,119 @@ letsencrypt() {
   fi
 }
 
+configure_pma() {
```

Comment:
I think you should also split this function into multiple smaller functions and use the correct indentation of `2` spaces.

Reply:
Done